libtailscale/store.go

```diff
@@ -23,6 +25,25 @@ func newStateStore(appCtx AppContext) *stateStore {
 	}
 }
 
+func (s *stateStore) All() iter.Seq2[ipn.StateKey, []byte] {
+	rawJSON := s.appCtx.GetStateStoreKeysJSON()
+	var keys []string
+	if err := json.Unmarshal([]byte(rawJSON), &keys); err != nil {
+		return func(yield func(ipn.StateKey, []byte) bool) {}
+	}
+	return func(yield func(ipn.StateKey, []byte) bool) {
+		for _, k := range keys {
+			blob, err := s.ReadState(ipn.StateKey(k))
```

Comment:
keys are prefixed with `statestore-` in the actual prefs (per `prefKeyFor`). should we remove that prefix here?